docs/general/administration/migrate.md

```diff
@@ -5,9 +5,15 @@ title: Migrating
 
 # Migrating
 
-It is possible to migrate your system to another system by using environment variables.
-It's possible to do this via the command line or by using Docker environment variables.
-To read more, see the [Configuration](/docs/general/administration/configuration) page.
+This page explains how to migrate from Emby to Jellyfin as well as how to migrate a Jellyfin installation.
```

Comment:
First of all, thanks for reviewing all changes. 

My bad, I thought it was justified because about half of the entire page was about migrating from Emby to Jellyfin. Should I replace "Emby" by "other media servers"?

Reply:
I think a sentence like "This page explains how to migrate Jellyfin installations." could apply both to Jellyfin>Jellyfin and Other>Jellyfin migrations.

Reply:
In my ears that sounds like "from jellyfin to X" only. I fixed all the other things and included a slightly different wording; maybe that one sounds fine for both of us.